src/Components/SysTable/constants.js

```diff
@@ -120,19 +120,14 @@ export const mergedColumns = (defaultColumns) => {
             undefined : {
                 ...(isStringCol ? { key: column } : column),
                 ...defaultColumn,
-                ...(column.key === 'groups'
-                    ? {
-                        renderFunc: (value) =>
-                            isEmpty(value?.groups) ? (
-                                'N/A'
-                            ) : (
-                            //TODO: replace a tag with Link component
-                                <a href={`/insights/inventory/groups/${value?.groups[0].id}`}>
-                                    {
-                                        value?.groups[0].name // currently, one group at maximum is supported
-                                    }
-                                </a>
-                            )
+                ...(column.key === 'groups' ?
+                    {
+                        renderFunc: (groups) => isEmpty(groups) ?
+                            'N/A'
+                            :
+                            <InsightsLink to={`/insights/inventory/groups/${groups[0].id}`}>
```

Comment:
If you want to ensure that in any case wether it is shown in inventory or in another app via fed mods, you should be able to use the `app` prop instead of using a full path.

So it would be: 
```js
<InsightsLink to={`/groups/${groups[0].id}`} app="inventory">
```

See https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/doc/insights_link.md#usage

Reply:
Thanks @bastilian.  I've fixed that up and confirmed it works as expected.